data/json/monsters/slimes.json

```diff
@@ -203,7 +203,7 @@
     "fear_triggers": [ "FIRE" ],
     "death_function": {
       "message": "The %s's body dissolves into acid.",
-      "effect": { "id": "death_acid", "hit_self": true },
+      "effect": { "id": "death_slimespring", "min_level": 2 },
```

Comment:
I'm looking at the spells and since the spells have no variance between the minimum and maximum values how is the `"min_level"` differences on the monster json having any effect?

Reply:
Guilt is a *face spell* and the `min_level` property determines the actual spell level in-game. In the example you reviewed above `death_slimespring` is a fake spell and is defined [here](https://github.com/matshou/Cataclysm-DDA/blob/8111ad0fe7d166f9d6541a331c135a392530a2b8/data/json/monster_special_attacks/monster_deaths.json#L575). You can see [here](https://github.com/matshou/Cataclysm-DDA/blob/8111ad0fe7d166f9d6541a331c135a392530a2b8/src/magic.cpp#L2397-L24248) how a fake spell gets converted to a *real spell* entry and how the `min_level` value is being used. Once the fake spell has been converted to a real spell it's level is then used in `spell_effect::guilt` function to determine the strength of guilt applied on the player. 

I think it's convenient for developers to define the strength of guilt directly from monsters json entries instead of defining custom fake spells which had to be done previously and would clutter the `monster_deaths.json` with entries.